test/test04/test04.m

```diff
@@ -0,0 +1,35 @@
+% Test to exersize the function SBE19Parse 
+% this function parses Sea-Bird DataProcessor generated files
+
+%sample_data = SBE19Parse({'test\test04\sbe16example.cnv'}, 'timeseries');
+%sample_data.meta.instrument_model
+
+
+%% Test 1: parse SBE19 file
+sd = SBE19Parse({'test\test04\Test 4550.cnv'}, 'timeseries');
+assert(strcmp(sd.meta.instrument_model,'SBE19plus'), 'Failed: Parse SBE19 file')
+
+
+%% Test 2: parse SBE9 file
+sd = SBE19Parse({'test/test04/SBE9_example.cnv'}, 'timeseries');
+assert(strcmp(sd.meta.instrument_model,'SBE 9'), 'Failed: Parse SBE9 file')
```

Comment:
Might keep the strrep() so that we get 'SBE9' instead of 'SBE 9' for consistency like Simon suggested.